buildbot/configure.py

```diff
@@ -178,6 +178,8 @@ def do_configure(args):
         "-DLLVM_ENABLE_PROJECTS={}".format(llvm_enable_projects),
         "-DSYCL_BUILD_PI_HIP_PLATFORM={}".format(sycl_build_pi_hip_platform),
         "-DLLVM_BUILD_TOOLS=ON",
+        "-DLLVM_ENABLE_ZSTD=ON",
```

Comment:
I think we shouldn't turn on these by default? We should just turn on them in `if args.ci_defaults:`

Reply:
The idea behind turning them on by default is to build the compiler with device image compression support if the user has zstd-dev package installed. If zstd is not found, there shouldn't be any build error.

Reply:
there are configurate failures.

Reply:
CMake Error at lib/Support/CMakeLists.txt:327 (get_property):

  get_property could not find TARGET zstd::libzstd_static.  Perhaps it has

  not yet been created.



CMake Error at lib/Support/CMakeLists.txt:330 (get_property):

  get_property could not find TARGET zstd::libzstd_static.  Perhaps it has

  not yet been created.

Reply:
yeah local build is cooked for me, we need to revert the on by zstd default part at least

Reply:
PR to temporarily turn off zstd by default: https://github.com/intel/llvm/pull/15833